app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Service Catalog
+subtitle: Track every service across your architecture
+content-type: explanation
+---
+
+{{site.konnect_saas}}'s Service Catalog offers a comprehensive catalog of all services running in your organization. This catalog gives you insight into your organization's services and their dependencies. It includes both {{site.base_gateway}} services and external integration resources.
+
+<!-- vale off-->
+{% mermaid %}
+graph LR
+  %% Define styles for icons and nodes
+  classDef resource fill:#E6D0F0
+  classDef konnect fill:#d0e5f3
+  classDef serviceCatalog fill:#e1f7d5
+  %% Define the resources
+  ExternalIntegration1[External Integration 1]:::resource
+  ExternalIntegration2[External Integration 2</div>]:::resource
+  GatewayManager[<div style="text-align:center;"><img src="/assets/images/logos/konglogo-gradient-secondary.svg" style="max-height:50px; max-width:50px; display:block; margin:0 auto;" class="no-image-expand"/> Gateway Manager</div>]:::resource
```

Comment:
```suggestion
  ExternalIntegration1(External Integration 1):::resource
  ExternalIntegration2(External Integration 2</div>):::resource
  GatewayManager(<div style="text-align:center;"><img src="/assets/images/logos/konglogo-gradient-secondary.svg" style="max-width:25px; display:block; margin:0 auto;" class="no-image-expand"/> Gateway Manager</div>):::resource
```
Rounded corner shapes. 

You don't have to accept my exact size, 50px was just an example. I think "display:block" might not be doing anything here either, it just gets ignored if you inspect the element via chrome tools.